src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -1,517 +1,524 @@
-
-        /// <summary>
-        /// Configures the Azure App Configuration provider to use the provided Key Vault configuration to resolve key vault references.
-        /// </summary>
-        /// <param name="configure">A callback used to configure Azure App Configuration key vault options.</param>
-        public AzureAppConfigurationOptions ConfigureKeyVault(Action<AzureAppConfigurationKeyVaultOptions> configure)
-        {
-            var keyVaultOptions = new AzureAppConfigurationKeyVaultOptions();
-            configure?.Invoke(keyVaultOptions);
-
-            if (keyVaultOptions.Credential != null && keyVaultOptions.SecretResolver != null)
-            {
-                throw new InvalidOperationException($"Cannot configure both default credentials and secret resolver for Key Vault references. Please call either {nameof(keyVaultOptions.SetCredential)} or {nameof(keyVaultOptions.SetSecretResolver)} method, not both.");
-            }
-
-            _adapters.RemoveAll(a => a is AzureKeyVaultKeyValueAdapter);
-            _adapters.Add(new AzureKeyVaultKeyValueAdapter(new AzureKeyVaultSecretProvider(keyVaultOptions)));
-
-            IsKeyVaultRefreshConfigured = keyVaultOptions.IsKeyVaultRefreshConfigured;
-            IsKeyVaultConfigured = true;
-            return this;
-        }
-
-        /// <summary>
-        /// Provides a way to transform settings retrieved from App Configuration before they are processed by the configuration provider.
-        /// </summary>
-        /// <param name="mapper">A callback registered by the user to transform each configuration setting.</param>
-        public AzureAppConfigurationOptions Map(Func<ConfigurationSetting, ValueTask<ConfigurationSetting>> mapper)
-        {
-            if (mapper == null)
-            {
-                throw new ArgumentNullException(nameof(mapper));
-            }
-
-            _mappers.Add(mapper);
-            return this;
-        }
-
-        /// <summary>
-        /// Configure the provider behavior when loading data from Azure App Configuration on startup.
-        /// </summary>
-        /// <param name="configure">A callback used to configure Azure App Configuration startup options.</param>
-        public AzureAppConfigurationOptions ConfigureStartupOptions(Action<StartupOptions> configure)
-        {
-            configure?.Invoke(Startup);
-            return this;
-        }
-
-        private static ConfigurationClientOptions GetDefaultClientOptions()
-        {
-            var clientOptions = new ConfigurationClientOptions(ConfigurationClientOptions.ServiceVersion.V2023_10_01);
-            clientOptions.Retry.MaxRetries = MaxRetries;
-            clientOptions.Retry.MaxDelay = MaxRetryDelay;
-            clientOptions.Retry.Mode = RetryMode.Exponential;
-            clientOptions.AddPolicy(new UserAgentHeaderPolicy(), HttpPipelinePosition.PerCall);
-
-            return clientOptions;
-        }
-    }
-}
+﻿// Copyright (c) Microsoft Corporation.

```

Comment:
Why is every single line of this file showing up oin the diff?

Reply:
Discussed offline, line endings were inconsistent for this file when merged into main from another PR